rider/serializer.py

```diff
@@ -8,15 +8,12 @@ class riderSerializer(serializers.ModelSerializer):
     class Meta:
         model = Rider
 
-    def restore_object(self, attrs, instance=None):
-        """
-        Create or update a new snippet instance.
-        """
-        if instance:
-            # Update existing instance
-            instance.push_id = attrs['push_id']
-            instance.os = attrs['os']
-            return instance
+    def update(self, instance, attrs):
+        #Update existing instance
+	instance.push_id = attrs['push_id']
```

Comment:
nope, but it does not appear to be in use anywhere server side. Also, most entries have that field empty